stdlib/zipfile.pyi

```diff
@@ -245,7 +253,13 @@ if sys.version_info >= (3, 8):
         def __init__(self, root: ZipFile | StrPath | IO[bytes], at: str = "") -> None: ...
         if sys.version_info >= (3, 9):
             def open(
-                self, mode: _ReadWriteBinaryMode = "r", *args: Any, pwd: bytes | None = None, **kwargs: Any
+                self,
+                mode: _ReadWriteBinaryMode = "r",
+                *args: Any,
+                pwd: bytes | None = None,
+                encoding: str | None = None,
+                force_zip64: bool = ...,
+                **kwargs: Any,
```

Comment:
I just checked and it seems that `encoding` is only supported on Python 3.10+, so we would need another branch for that.

I don't see `force_zip64` supported in any version?

Reply:
Yes, `force_zip64` don't supported in versions >= 3.9.

`encoding` don't supported in versions < 3.9
`encoding` can be supplied as a kwarg in py3.9
Changed in version 3.10.10: The `encoding` parameter can be supplied as a positional argument